### ISSUE TITLE
Release plugin support

### ISSUE THREAD

**Issue by timothyb89** [monasca/pr-bot#9]
It would be nice to have (in addition to mutation/update plugin) support for release plugin. These would perform whatever tasks are needed to release a new version. Some examples:
 * PR to helm git source to bump chart version, user can specify a new version number or 'major', 'minor', or 'patch' to increment the respective field by +1 set others to 0
   * CI in monasca-helm will pick up the change and automatically release once the PR is approved
 * PR to docker git source to trigger rebuild
   * for images using dbuild and the master-datestamp versioning policy, just put up an empty commit with `!push <module>` in the commit message
   * for for semver, follow helm semantics
   * once approved, images are rebuild and pushed to docker hub, pr-bot already notifies about the change
